common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -1,5 +1,5 @@
 $sequence--border-color: #C8C8C8;
-
+$link-color: rgb(26, 161, 222);
```

Comment:
@marcotuts can we access the variables from _variables.scss in this file ? i'm not sure about that.